docs/src/testing/test-collection.md

```diff
@@ -93,3 +93,15 @@ The package structure of tests makes it easy!
 In both of the above examples, you can
 make the functions from `tests/common/utils.cairo` available in `tests/test_contract.cairo` 
 by using a relative import: `use super::common::utils;`.
+
+### How Contracts Are Collected
+
+When you call `snforge test`, one of the things that Forge does is that it calls Scarb, particularly `scarb build`.
+Scarb then builds all contracts you're using and stores them inside the `target/{current_profile}` directory
```

Comment:
```suggestion
It makes Scarb build all contracts from your package and save them to the `target/{current_profile}` directory
```

Reply:
It's not strictly true since you can build external contracts as well

Reply:
Yup, but then we can mention about it in the next section and it's all clear